vlib/v/checker/tests/method_op_alias_err.out

```diff
@@ -0,0 +1,28 @@
+vlib/v/checker/tests/method_op_alias_err.vv:4:1: error: both sides of an operator must be the same type
+    2 | type Foo2 = string
+    3 | 
+    4 | fn (f Foo) + (f1 Foo2) Foo2 {
+      | ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    5 |     return Foo2(f + f1)
+    6 | }
+vlib/v/checker/tests/method_op_alias_err.vv:5:19: error: infix expr: cannot use `string` (right expression) as `string`
```

Comment:
this error is fail!

Reply:
it should be something like; `recusive function call` or something. @JalonSolov what do you think?

Reply:
Again, I don't see this in his changes.  This one definitely needs to be fixed, but in a separate PR.

Reply:
Yes